Reduce merge conflicts in VerifyCACerts

rgithubli opened the pull request:
Similar to https://github.com/corretto/corretto-21/pull/6.

Forward port of https://github.com/corretto/corretto-17/commit/fe7ab0b99c122da3dd5d7d62e309ef90d32ba864.

The port is NOT clean since 21 has more 3 certs than 17. The `AMAZON_CA_CERT_COUNT ` is 54 (instead of 51) and its `AMAZON_CA_FINGERPRINT_MAP ` has the 3 more certs added.

rgithubli committed:
Reduce merge conflicts in VerifyCACerts

mrserb commented:
probably this should be integrated first:
https://github.com/corretto/corretto-jdk/pull/95#pullrequestreview-1515561519

mrserb commented:
We also need this in 21

mrserb commented:
That one is pushed, please update/integarete this

mrserb commented:
we also need this patch in the corretto-8

rgithubli committed:
Merge branch 'develop' into ca-cert

rgithubli committed:
Remove expired certs

Merge 2baaf3d2bab64aa8ae93c7fad94873924fd94bb8 and bde467f01d76631e705307bb020ec474a0490f72
